chicagoland.pandemicresponsecommons.org/manifests/hatchery/jupyter-covid19.yaml

```diff
@@ -6,9 +6,9 @@ services:
          - ${DATA_VOLUME}:/data
          - ${USER_VOLUME}:/home/jovyan/pd
       environment:
-        - NAMESPACE=default
-        - HOSTNAME=chicagoland.pandemicresponsecommons.org
-        - FRAME_ANCESTORS=https://chicagoland.pandemicresponsecommons.org
+        - NAMESPACE=qa-covid19
+        - HOSTNAME=qa-covid19.planx-pla.net
+        - FRAME_ANCESTORS=https://qa-covid19.planx-pla.net
```

Comment:
uh oh, this should be added to environment specific fields

Reply:
Correct. Let me create a PR..